src/Tribe/Tickets_Handler.php

```diff
@@ -310,6 +314,13 @@ private function generate_filtered_attendees_list( $event_id ) {
 				if ( 'check_in' === $column_id && 1 == $single_item[ $column_id ] ) {
 					$row[ $column_id ] = esc_html__( 'Yes', 'event-tickets' );
 				}
+
+				// Special handling for new human readable id
+				if ( 'attendee_id' === $column_id ) {
+					$ticket_unique_id  = get_post_meta( $single_item[ $column_id ], '_unique_id', true );
```

Comment:
Can we make sure we have the variable to prevent notices? On `$single_item[ $column_id ]`